src/AppBoard/Board.java

```diff
@@ -0,0 +1,32 @@
+package AppBoard;
+
+public class Board {
+    private User user;
+    private Message message;
+
+    User ion = new User("ionmatei@gmail.com", "Ion");
+    User paul = new User("paulmargaritiu@gmail.com", "Paul");
+    User matei = new User("mateimatei@gmail.com", "Matei");
+    User maria = new User("mariamaria@gmail.com", "Maria");
```

Comment:
mai potrivit ar fi sa transformi obiectele User de mai sus intr-o colectie, in care sa poti adauga si mai tarziu useri